kroma-devnet/devnet/__init__.py

```diff
@@ -205,6 +271,38 @@ def devnet_l1_genesis(paths):
     finally:
         geth.terminate()
 
+# [Kroma: START]
+def e2e_devnet_l1_genesis(paths):
```

Comment:
Most code blocks of this function is duplicate. How about unifying two functions and add some arguments to use in both cases?